app.rb

```diff
@@ -0,0 +1,168 @@
+class App
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def show_menu
+    puts "Welcome to School Library App!\n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
```

Comment:
- [ ] Your console app should behave in the same way as it is in the video example, if the user chooses an option kindly consider returning the output but don't exit the app until the user selects option 7. as for now the app exit after returning  anything from the other options 

![Peek 2022-08-06 15-28](https://user-images.githubusercontent.com/25227022/183253145-d86af0f2-333d-4832-b971-addb824acf7f.gif)